zarr/store/zipfile.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+
+from zarr.store.base import ArrayStore
+
+
+class ZipFileStore(ArrayStore):
+
+    def __init__(self, path):
+        self._path = path
+        # TODO
+
+    @property
+    def meta(self):
+        pass
+
+    @property
+    def data(self):
+        pass
+
+    @property
+    def attrs(self):
+        pass
+
+    @property
+    def cbytes(self):
+        pass
+
+    @property
+    def initialized(self):
+        pass
```

Comment:
That was my meaning, yes.  Though the interface looks manageable enough to not be a big deal to implement.  

I am still motivated to find a way to support completely foreign MutableMappings.  I wonder if we could manage this by creating an `ArrayStore` that consumes a couple of MutableMappings, perhaps one in-memory dict or zipfile that manages metadata and another that just manages bulk data out on some large data store.  We would pass around the small mapping which would contain a pointer to the larger one.